lib/schema_plus/foreign_keys.rb

```diff
@@ -1,4 +1,5 @@
 require 'schema_plus/core'
+require 'schema_plus_compatibility'
```

Comment:
I'm using `tables_without_deprecation` in lib/schema_plus/foreign_keys/middleware/dumper.rb, so it cannot be a dev dependency.